llvm/test/Bitcode/clone-local-types.ll

```diff
@@ -0,0 +1,43 @@
+; RUN: llvm-as < %s -o %t0
+; RUN: llvm-dis %t0 -o - | FileCheck %s
+
```

Comment:
might be worth a comment describing the purpose of this test (some info from the commit message, probably)